server/src/main/java/org/candlepin/model/PoolCurator.java

```diff
@@ -2006,4 +2026,93 @@ public void clearPoolSourceEntitlementRefs(Iterable<String> poolIds) {
 
         return output;
     }
+
+    /**
+     * Fetches a collection of active pool IDs mapped to a given owner. If no such pool
+     * are present, an empty collection will be returned.
+     *
+     * @param ownerId
+     *  The ID of the owner for which to fetch active Pool IDs
+     *
+     * @return
+     *  A collection of active Pool IDs belonging to the given owner.
+     */
+    public Collection<String> getActivePoolByOwnerId(String ownerId) {
```

Comment:
This should be named "getActivePool*Ids*ByOwnerId", as it is fetching multiple pool IDs, not a single pool.

Reply:
Removed.